EventStore.API/Services/ISchedulingStatisticsCalculator.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using EventStore.API.DTOs;
+
+namespace EventStore.API.Services
+{
+    public interface ISchedulingStatisticsCalculator
+    {
+        public SchedulingStatisticsDto GetSchedulingStatistics();
+        public StepsStatisticsDto GetStepsStatistics();
+        public IDictionary<int, TimeSpan> GetAgeStatistics();
+        public SuccessStatisticsDto GetSuccessStatistics();
+        public SchedulingTimeDto GetSchedulingTimeStatistics();
+        public IDictionary<int, int> GetStepLeavingRateStatistics();
+        public IDictionary<int, TimeSpan> GetStepDurationStatistics();
```

Comment:
Remove public. Same situation in ISchedulingStatisticsService.cs.

Reply:
I have to leave them public in order to properly test every statistic method. For more info see Scheduling service unit tests.